doc/source/man.rst

```diff
@@ -32,206 +32,217 @@ rho performs four major tasks:
 
-rho profile edit --name=myprofile --auth server1creds server2creds
+``rho profile edit --name=myprofile --auth server1creds server2creds``
 
-You can use ``rho profile show --name=myprofile`` to make sure that the profile was properly edited.
+**TIP:** After editing a network profile, use the ``rho profile show`` command to review those edits.
 
-Listing and Showing Profiles
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Listing and Showing Network Profiles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The ``list`` commands lists the details for all configured profiles. The output includes the IP ranges, auth credentials, and ports for the profile.
+The ``rho profile list`` command returns the details for all configured network profiles. The output of this command includes the hosts, IP addresses, or IP ranges, the authentication profiles, and the ports that are configured for each network profile.
 
 **rho profile list [--vault=** *vault_file* **]**
 
 ``--vault=vault_file``
 
-  This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path and file name of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
 
-The ``rho profile show`` command is the same as the ``rho profile list`` command, except that it returns details for a single specific profile. This is a handy command to verify edits to a profile.
+The ``rho profile show`` command is the same as the ``rho profile list`` command, except that it returns details for a single specified network profile.
 
 **rho profile show --name=** *profile* **[--vault=** *vault_file* **]**
 
 ``--name=profile``
 
-  This argument gives the profile to display.
+  Required. Contains the network profile to display.
 
 ``--vault=vault_file``
 
-  This contains the path and filename of the file containing the vault password. If this option is used the file should be limited in access on the system.
+  Contains the path and file name of the file that contains the vault password. Because the encrypted Rho data could contain sensitive information, make sure that this vault password file is stored in a location that has limited access.
 
-Deleting Profiles
-~~~~~~~~~~~~~~~~~
+Clearing Network Profiles
+~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Any or all profiles can be deleted using the ``clear`` subcommand.
+As the network infrastructure changes, it might be necessary to delete some network profiles. Use the ``rho profile clear`` command to delete network profiles.
 
 **rho profile clear --name=** *name* **| --all [--vault=** *vault_file* **]**
```

Comment:
```
 **rho profile clear (--name=** *name* **| --all) [--vault=** *vault_file* **]**
```

Reply:
Change made:
**rho profile clear (--name=** *name* **| --all) [--vault=** *vault_file* **]**
Also added the "Mutually exclusive with the X option." statements to --name and --all descriptions.